src/DocumentFormat.OpenXml/OpenXmlPartReader.cs

```diff
@@ -44,17 +44,20 @@ private OpenXmlPartReader(bool readMiscNodes)
         }
 
         /// <summary>
-        /// Initializes a new instance of the OpenXmlPartReader class using the supplied OpenXmlPart class.
+        /// Initializes a new instance of the OpenXmlPartReader class using the supplied OpenXmlPart and Boolean values.
         /// </summary>
         /// <param name="openXmlPart">The OpenXmlPart to read.</param>
-        public OpenXmlPartReader(OpenXmlPart openXmlPart) : this()
+        /// <param name="readMiscNodes">Specify false to indicate to the reader to skip all miscellaneous nodes. The default value is false.</param>
+        /// <param name="ignoreWhitespace">Specify true to indicate to the reader to ignore insignificant white space. The default value is true.</param>
+        public OpenXmlPartReader(OpenXmlPart openXmlPart, bool readMiscNodes = false, bool ignoreWhitespace = true)
```

Comment:
You'll need to add a new constructor here that takes in the new option. This would be a binary compat breaking change as-is.

Reply:
Restorted the old constructors and add a new one